routes/minecraft/ranks.js

```diff
@@ -12,6 +12,20 @@ module.exports = function (app) {
             res.json(ranks);
         })
     });
+    
+    app.get('/mc/players/:rank', (req, res) => {
+        MinecraftRank.find({ name: req.params.rank.toLowerCase() }).limit(1).exec((err, ranks) => {
+            var rank = ranks[0];
+            
+            if (!rank) {
+                res.status(401).json({ notFound: true });
```

Comment:
Why are you returning 401?

Reply:
im copying the other endpoints, the one just below it uses 401 for notfound

Reply:
HTTP 401 is very different from HTTP 404.

HTTP 401: Unauthorized - you do not have the proper permission to view a resource
HTTP 404: Not Found - the resource is unavailable or does not exist

I’ll open a PR later today to bring the rest up to code and uniform. You should use 404 here

Reply:
Exactly what I thought! Thank you. Someone needs to teach the API developers a lesson HTTP, haha.